tests/admin/valuesets/valueSetCreation.spec.ts

```diff
@@ -0,0 +1,347 @@
+    await page.getByRole("textbox", { name: "Name *" }).fill(uniqueName);
+    await page.getByRole("textbox", { name: "Slug *" }).fill(uniqueSlug);
+
+    // Add at least one include rule (required)
+    await page
+      .getByRole("button", { name: "+ Add Rule", exact: true })
+      .first()
+      .click();
+
+    // Select coding system
+    await page.getByRole("combobox", { name: "System" }).first().click();
+    await page.getByRole("option", { name: "SNOMED CT" }).click();
+
+    // Add a concept
+    await page.getByRole("button", { name: "+ Add Concepts" }).first().click();
+    await page
+      .getByRole("textbox", { name: "Code *" })
+      .first()
+      .fill("73211009");
+
+    // Set status
+    await page.getByRole("combobox", { name: "Status" }).click();
+    await page.getByRole("option", { name: "Active" }).click();
+
+    // Save the ValueSet
+    await page.getByRole("button", { name: "Save" }).click();
+
+    // Verify success with proper waiting
+    await expect(page.getByText("ValueSet created successfully")).toBeVisible({
+      timeout: 10000,
+    });
+
+    await expect(page.getByRole("cell", { name: uniqueName })).toBeVisible();
+  });
+
+  test("should create a valueset with all fields", async ({ page }) => {
+    const uniqueSuffix = faker.string.alphanumeric(8).toLowerCase();
+    const uniqueName = `All Fields ValueSet ${uniqueSuffix}`;
+    const uniqueSlug = `all-fields-${uniqueSuffix}`;
+    const description = faker.lorem.sentence();
+
+    // Fill all basic fields
+    await page.getByRole("textbox", { name: "Name *" }).fill(uniqueName);
+    await page.getByRole("textbox", { name: "Slug *" }).fill(uniqueSlug);
+    await page.getByRole("textbox", { name: "Description" }).fill(description);
+
+    // Set status
+    await page.getByRole("combobox", { name: "Status" }).click();
+    await page.getByRole("option", { name: "Draft" }).click();
+
+    // Add include rule with concepts
+    await page
+      .getByRole("button", { name: "+ Add Rule", exact: true })
+      .first()
+      .click();
+    await page.getByRole("combobox", { name: "System" }).first().click();
+    await page.getByRole("option", { name: "LOINC" }).click();
+
+    // Add multiple concepts
+    await page.getByRole("button", { name: "+ Add Concepts" }).first().click();
+    await page.getByRole("textbox", { name: "Code *" }).first().fill("1558-6");
+
+    await page.getByRole("button", { name: "+ Add Concepts" }).first().click();
+    const secondConcept = page.getByRole("textbox", { name: "Code *" }).nth(1);
+    await secondConcept.fill("2339-0");
+
+    // Save and verify
+    await page.getByRole("button", { name: "Save" }).click();
+
+    await expect(page.getByText("ValueSet created successfully")).toBeVisible({
+      timeout: 10000,
+    });
+    await expect(page.getByRole("cell", { name: uniqueName })).toBeVisible();
+  });
+
+  test("should create valueset with include rules - any system with concept only", async ({
```

Comment:
- You already tested the same in the above two tests, so why are we adding it here again. If needed check for UCUM code, that is only thing missed in the concept only creation